rhel7/fixes/bash/grub2_enable_fips_mode.sh

```diff
@@ -16,19 +16,23 @@ package_command install dracut-fips
 
 dracut -f
 
-if [ -e /sys/firmware/efi ]; then
-	BOOT=`df /boot/efi | tail -1 | awk '{print $1 }'`
-else
-	BOOT=`df /boot | tail -1 | awk '{ print $1 }'`
-fi
-
 # Correct the form of default kernel command line in /etc/default/grub
 if ! grep -q "^GRUB_CMDLINE_LINUX=\".*fips=1.*\"" /etc/default/grub;
 then
   # Append 'fips=1' argument to /etc/default/grub (if not present yet)
   sed -i "s/\(GRUB_CMDLINE_LINUX=\)\"\(.*\)\"/\1\"\2 fips=1\"/" /etc/default/grub
```

Comment:
Quite a lot of double quote escaping could have been avoided by changing the outer quotes to single ones. Could you please do us a favor and expand the PR to go for single quotes whenever it is applicable in this script?